time stamp
float32[] pose_1
float32[] pose_2
float32[] point_cloud_1
float32[] point_cloud_2
